firmware/dev/f4dev/cubemx/f4dev.ioc.md5

```diff
@@ -0,0 +1 @@
+80041d9bb87dbdcb83a1ac6a7824aa29
```

Comment:
Probably another artifact of merge conflicts - if you can undo this diff that would be awesome!